bigtable-dataflow-parent/bigtable-hbase-beam/src/main/java/com/google/cloud/bigtable/beam/CloudBigtableScanConfiguration.java

```diff
@@ -164,17 +229,67 @@ public CloudBigtableScanConfiguration build() {
           scan = new Scan();
         }
         ReadRowsRequest.Builder builder = Adapters.SCAN_ADAPTER.adapt(scan, readHooks);
-        request = readHooks.applyPreSendHook(builder.build());
+        request =  StaticValueProvider.of(readHooks.applyPreSendHook(builder.build()));
```

Comment:
Is `scan` going to be a value provider in the future?

Reply:
If so, then we need yet another implementation `ValueProvider< ReadRowsRequest>` similar to RequestWithTableNameValueProvider.

Reply:
I have found no reason to have ValueProvider\<Scan\>.